nodejs/eks/package.json

```diff
@@ -18,7 +18,8 @@
         "netmask": "^1.0.6",
         "@types/js-yaml": "^3.12.0",
         "js-yaml": "^3.13.0",
-        "which": "^1.3.1"
+        "which": "^1.3.1",
+        "https-proxy-agent": "^5.0.0"
     },
```

Comment:
Weird - we removed usage of `node-fetch` from the code - but I don't even see it listed here.  How was this working before?  (did we just so happen to pick it up transitively from another package)?